src/openzaak/components/documenten/api/viewsets.py

```diff
@@ -981,3 +983,27 @@ def destroy(self, request, *args, **kwargs):
         if settings.CMIS_ENABLED:
             raise CMISNotSupportedException()
         return super().destroy(request, *args, **kwargs)
+
+
+class ReservedDocumentViewSet(mixins.CreateModelMixin, viewsets.GenericViewSet):
+    queryset = ReservedDocument.objects.all()
+    serializer_class = ReservedDocumentSerializer
+
+    @extend_schema(
+        summary="Reserveer een documentnummer",
+        description=(
```

Comment:
Since this endpoint is not part of the Documenten API standard, it has to be marked as "experimental" in the API spec and listen in `docs/api/experimental.rst` as a experimental feature. Could you add that?
```suggestion
        from openzaak.utils.help_text import mark_experimental
        ...
        description=mark_experimental(
```